bug_report_page.php

```diff
@@ -201,11 +201,8 @@
 $t_show_os_version = $t_show_profiles && in_array( 'os_version', $t_fields );
 $t_show_resolution = in_array( 'resolution', $t_fields );
 $t_show_status = in_array( 'status', $t_fields );
-$t_show_tags =
-	in_array( 'tags', $t_fields ) &&
-	access_has_global_level( config_get( 'tag_view_threshold', /* default */ null, /* user */ null, $t_project_id ) );
 $t_can_attach_tag =
```

Comment:
I would rename this variable to $t_show_tags since it doesn't necessarily ability to add tags but whether they should be displayed or not.